code/game/gamemodes/miniantags/abduction/abduction.dm

```diff
@@ -149,32 +149,36 @@
 	greet_scientist(scientist,team_number)
 	update_abductor_icons_added(scientist)
 
-/datum/game_mode/abduction/proc/greet_agent(datum/mind/abductor,team_number)
+/datum/game_mode/abduction/proc/greet_agent(datum/mind/abductor, team_number)
 	var/datum/objective/stay_hidden/O = new
 	abductor.objectives += O
 	abductor.objectives += team_objectives[team_number]
 	var/team_name = team_names[team_number]
 
 	SEND_SOUND(abductor.current, sound('sound/ambience/antag/abductors.ogg'))
-	to_chat(abductor.current, "<span class='notice'>You are an agent of [team_name]!</span>")
-	to_chat(abductor.current, "<span class='notice'>With the help of your teammate, kidnap and experiment on station crew members!</span>")
-	to_chat(abductor.current, "<span class='notice'>Use your stealth technology and equipment to incapacitate humans for your scientist to retrieve.</span>")
-	to_chat(abductor.current, "<span class='motd'>For more information, check the wiki page: ([GLOB.configuration.url.wiki_url]/index.php/Abductor)</span>")
+	var/list/messages = list()
+	messages.Add("<span class='notice'>You are an agent of [team_name]!</span>")
+	messages.Add("<span class='notice'>With the help of your teammate, kidnap and experiment on station crew members!</span>")
+	messages.Add("<span class='notice'>Use your stealth technology and equipment to incapacitate humans for your scientist to retrieve.</span>")
+	messages.Add("<span class='motd'>For more information, check the wiki page: ([GLOB.configuration.url.wiki_url]/index.php/Abductor)</span>")
```

Comment:
```suggestion
	messages.Add("<span class='motd'>For more information, check the wiki page: [wiki_link("Abductor", "Abductor")]</span>")
```
Yesterday, @warriorstar-orion had their PR merged that introduces a wiki link formatter with https://github.com/ParadiseSS13/Paradise/pull/22504 that may work here and in the other places that have hardcoded wiki URLs. This might be worth giving a try? I would test this though to make sure it works.

Reply:
Considering there are a fair few of these on things I don't touch too, it might be needing a follow-up PR that atomically handles this change. I don't want to leave us half-and-half about it.

Reply:
That is understandable. I can agree that it would be a good PR for the future. Thanks for taking the time to consider it.